docs/source/conf.py

```diff
@@ -122,3 +122,8 @@ def setup(app):
                 obj.__name__ = name
 
     app.connect("autodoc-process-signature", process_signature)
+
+
+intersphinx_mapping = {
+    "graphviz": ("https://graphviz.readthedocs.io/en/stable/", None),
+}
```

Comment:
Noob question: what this `dict` is used for?

Reply:
With that, the cross links in the  `to_graph` function, i.e.

```rst
- :meth:`~graphviz.Digraph.render`: Save the graph to a file.
- :meth:`~graphviz.Digraph.view`: Open the graph in a viewer.
```

will actually link to to official documentation of `graphviz`.